recipe/meta.yaml

```diff
@@ -47,11 +47,6 @@ requirements:
 
 test:
   commands:
-    # The xraylib command does not work on Anaconda, as it would not be able
-    # to find the help text files. In my xraylib SDK for Windows, I rely on
-    # the registry to find them as the installer writes entries in it, but
-    # not sure how that would work here.
-    - xraylib  # [not win]
```

Comment:
@tschoonj, thanks for merging. Could you please elaborate on why the test was removed?

Reply:
I removed the xraylib command line utility in this release.

Reply:
Thank you for the information! We are picking it up in the nsls2forge channel (https://github.com/nsls-ii-forge/xraylib-feedstock/pull/3).

Reply:
Cool. Do keep in mind that changes in this release will probably impact your code: unacceptable arguments will now result in exceptions being thrown...

Reply:
Yes, we noticed 🙂, and made a couple of PRs to handle it. Thanks for the heads up!